pkg/conduit/entrypoint.go

```diff
@@ -71,6 +71,16 @@ func (e *Entrypoint) Serve(cfg Config) {
 	}
 }
 
+func deprecatedFlag(name string) bool {
```

Comment:
Nitpick: maybe you can turn this into a map, and just use the map:
```go
deprecatedFlags := map[string]bool{
    "pipelines.exit-on-error": true
}
if deprecatedFlags["something"] {
    ...
}
```

Reply:
@hariso good idea!